app/views/helpers/gmpInput.scala.html

```diff
@@ -52,13 +52,16 @@
         @if(labelHighlight) {<strong>}
         @if(elements.args.contains('_label)) {@elements.label}
         @if(labelHighlight) {</strong>}
-        @if(elements.args.contains('_hintText)) {
-            <span class="form-hint" id="@hintId">@elements.args.get('_hintText)</span>
-        }
         @if(elements.args.contains('_symbol)) {
             <span class="inline-fields">@elements.args.get('_symbol)</span>
         }
+        @if(elements.args.contains('_header)){
```

Comment:
I wonder if is better to move header above _symbol just in case or you checked paged where you used header and header is first on those pages?